Data Sync downstream reivew (#656)
aerogear/mobile-docs: modules/ROOT/pages/_partials/sync/js-client-getting-started.adoc

.Prerequisites
* You have Node.js and npm installed.

> * You have created an empty web project that supports ES6.  The link:https://webpack.js.org/guides/getting-started/[webpack getting started] project is a useful guide.

Reviewer: ```suggestion
* You have created an empty web project that supports ES6, for example, using the link:https://webpack.js.org/guides/getting-started/[webpack getting started] project.
```
